docs/self-hosted/quickstart.md

```diff
@@ -99,3 +99,14 @@ docker compose up
 :::
 
 Directus should now be available at http://localhost:8055 or http://127.0.0.1:8055.
+
+:::tip `SQLITE_CANTOPEN`
+
+If you see this error message when first running `docker compose up`, it means that your computer's user profile doesn't
+have the necessary ownership of the local `database` directory. This directory is connected to the docker container
+where Directus will run.
+
+On macOS/Linux, run `sudo chown $USER:$USER database` from the directory with your `docker-compose.yml` file to change
+the ownership to the current user.
+
+:::
```

Comment:
Wondering if it would be better to just include the folder creation in the initial instruction as a preventive measure, because the issue with permissions might happen with the other folders as well:

> Create a new empty folder on your Desktop called `directus`. Within this new folder, create the three empty folders `database`, `uploads`, and `extensions`.

Reply:
I like that a lot more. I'll apply the change.